website/docs/r/domain.html.markdown

```diff
@@ -276,18 +279,24 @@ resource "libvirt_domain" "my_machine" {
 }
 ```
 
-Note the following:
-* You can repeat the `console` block to create more than one console, in the same way
-that you can repeat `disk` blocks (see above)
-* The `target_type` is optional for the first console
-* All subsequent `console` blocks must specify a `target_type` of `virtio`
-* The `source_path` is optional for all consoles
+~> **Note well:**
+  <ul>
+    <li>You can repeat the `console` block to create more than one console, in the
+        same way that you can repeat `disk` blocks (see [above](#handling-disks)).</li>
+    <li>The `target_type` is optional for the first console.</li>
```

Comment:
I took a quick look at the code and I couldn't understand the default type. Do you know it? You wrote the code 😄